src/pages/trading-specification/components/_calculationCard.tsx

```diff
@@ -18,13 +18,13 @@ const CalculationCard = ({ data }: DlMarketTypeProps) => {
             </Header>
 
             {data.list.map((list, index) => (
-                <>
-                    <Header key={index} type="paragraph-2" weight="normal" as="p">
+                <React.Fragment key={index}>
```

Comment:
Do we have an `id` to use here instead of index? there was a discussion about this iteration rendering.

cc: @ali-hosseini-deriv

Reply:
we don't have `id`. `data.list` is having `details, icon` only